Changelog.md

```diff
@@ -0,0 +1,48 @@
+------------------------------------------------------
```

Comment:
did you write this file manually? If so, then I would rather not have it and try to fix my action that would generate it automatically.

Reply:
yes as every github project must have a changlog, I copied the previous commits and added my own, I never worked with automatic functions, I was taught to do it properly by hand.
I've never worked with an automatic function, I was taught to do it properly by hand, but if later you modify or delete it, that's fine too, but it already exists, that's the main thing. 
=)